example/tests/example_test.py

```diff
@@ -1,22 +1,60 @@
 import unittest
 
-from snapstack import Runner
+from snapstack import Plan, Step, Setup, Cleanup
 
 
 class ExampleTest(unittest.TestCase):
 
-    def test(self):
-        r = Runner(
+    def test_example(self):
+        '''
+        _test_example_
+
+        Basic test, which installs and tests and example
+        snap, after letting the Plan object build the default
+        snapstack.
+
+        Note: this will install snapstack on your system!
+
+        '''
+        example = Step(
             snap='example',  # Name of the snap
-            location='./tests/',  # Parent location for tests and files
-                                  # below. {local} is a magic string that
-                                  # points at the 'tests' directory.
+            script_loc='./tests/',  # Parent location for tests and files
+                                    # below. {local} is a magic string that
+                                    # points at the 'tests' directory.
             tests=['example.sh'],  # Test the snap
             files=['example.json'],  # Config files
-            base=None  # Don't override default Base.
+            snap_store=False  # Build from local source, rather than
```

Comment:
@coreycb Nothing actually uses it, so it didn't need to exist. I can add a stub, though.